docs/en/interfaces/formats.md

```diff
@@ -1020,6 +1020,49 @@ Example:
 }
 ```
 
+To use object name as column value you can use special setting [format_json_object_each_row_column_for_object_name](../operations/settings/settings.md#format_json_object_each_row_column_for_object_name).
+Examples:
+
+For output:
+```sql
+insert into function file('data.json', JSONObjectEachRow) select 'obj' as object_name, number from numbers(3) settings format_json_object_each_row_column_for_object_name='object_name'
+```
+
+File "data.json" will contain:
+```json
+{
+	"obj": {"number":"0"},
+	"obj": {"number":"1"},
+	"obj": {"number":"2"}
+}
```

Comment:
This JSON example looks not natural, because we have three rows with the same key and iirc duplicate keys are not allowed in valid JSON object. Is this intended to show this unique feature? If this is the case I'd like to see this as not the first example.

Reply:
You are right, this is a bad example, I will change it